dask/dataframe/core.py

```diff
@@ -1741,6 +1741,65 @@ def __repr__(self):
                                           name=key_split(self._name),
                                           task=len(self.dask))
 
+    @derived_from(pd.Series)
+    def rename(self, index=None, inplace=False, sorted_index=False):
+        """Alter Series index labels or name
+
+        Function / dict values must be unique (1-to-1). Labels not contained in
+        a dict / Series will be left as-is. Extra labels listed don't throw an
+        error.
+
+        Alternatively, change ``Series.name`` with a scalar value.
+
+        Parameters
+        ----------
+        index : scalar, hashable sequence, dict-like or function, optional
+            dict-like or functions are transformations to apply to
+            the index.
+            Scalar or hashable sequence-like will alter the ``Series.name``
+            attribute.
+        inplace : boolean, default False
+            Whether to return a new %(klass)s. If True then value of copy is
+            ignored.
+        sorted_index : bool, default False
+            In the case of a dict-like or function - if the input series has
```

Comment:
I prefer phrasing these as "what the keyword does" and then listing all the caveats. So something like

    sorted_index : bool, default False
        The transformed index is sorted, so the returned `Series` will have known divisons.
        This only applies to dict-like or function 'index' when the input series has known divisions.
        This should only be set to ``True``
        if you know that the mapping is monotonic. Dask checks that transformed divisions
        are monotonic, but cannot check all the values between divisions.